spec/views/pages/privacy_policy.html.slim_spec.rb

```diff
@@ -7,10 +7,19 @@
     render
   end
 
-  it 'includes a link to create a new account' do
+  it 'links to the system of record notices' do
+    pending 'having the correct link'
+
+    render
+
+    doc = Nokogiri::HTML(rendered)
+    expect(doc.xpath("//a[text()='systems of records notices']").first[:href]).to_not eq('#')
```

Comment:
this seems very specific / likely to break with page changes. Possible to look for the locale text instead of using xpath?